### PR TITLE
Support Django 4

### PR DESCRIPTION
**Pull request by dyve**
Fixes: #398

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 from .forms import TestForm
 
-DJANGO3 = get_version() >= "3"
+DJANGO_VERSION = int(get_version()[:1])
```
Comment (jieter) [inline tests/utils.py:6]: Why not use the package's `DJANGO_VERSION`?

**Review by jieter — APPROVED** [review]
Since this is fixed in django-bootstrap5, it would be nice to have a link to the commit implementing it over there.

This patch looks good in itself, I'd be ok merging it like this.

**Comment by dyve** [comment]
Django 4 support in django-bootstrap5 works differently because the renderers were rewritten for that package. Here's the link:
https://github.com/zostera/django-bootstrap5/pull/193

File: tests/utils.py (hunk 1 above)
Comment (dyve) [reply tests/utils.py]: I'll do that, good catch.

**Pull request merged**